samples/model-builder/create_hyperparameter_tuning_job_sample.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_sdk_create_hyperparameter_tuning_job_sample]
+from google.cloud import aiplatform
+
+from google.cloud.aiplatform import hyperparameter_tuning as hpt
+
+
+def create_hyperparameter_tuning_job_sample(
+    project: str,
+    location: str,
+    staging_bucket: str,
+    display_name: str,
+    container_uri: str,
+):
+    aiplatform.init(project=project, location=location, staging_bucket=staging_bucket)
+
+    worker_pool_specs = [
+        {
+            "machine_spec": {
+                "machine_type": "n1-standard-4",
+                "accelerator_type": "NVIDIA_TESLA_K80",
+                "accelerator_count": 1,
+            },
+            "replica_count": 1,
+            "container_spec": {
+                "image_uri": container_uri,
+                "command": [],
+                "args": [],
+            },
+        }
+    ]
+
+    custom_job = aiplatform.CustomJob(
+        display_name='custom_job',
+        worker_pool_specs=worker_pool_specs,
+    )
+
+    hpt_job = aiplatform.HyperparameterTuningJob(
+        display_name=display_name,
+        custom_job=custom_job,
+        metric_spec={
```

Comment:
We usually pass in some of these as parameters to the function but I'm not sure how to include things like parameter specs without cluttering the parameter space.

Reply:
@dizcology do you have guidance or is this okay?